packages/components/src/pages/VsCodeExtension.vue

```diff
@@ -973,6 +983,14 @@ export default {
         const { filters, expandedPackages } = state;
         if (filters) this.$store.commit(SET_FILTER, deserializeFilter(filters));
 
+        const base64EncodedFilters = base64UrlEncode(JSON.stringify({ filters }));
+        const selectedFilter =
+          this.$store.state.savedFilters &&
+          this.$store.state.savedFilters.find(
+            (savedFilter) => savedFilter.state === base64EncodedFilters
+          );
+        if (selectedFilter) this.$store.commit(SET_SELECTED_FILTER, selectedFilter);
```

Comment:
I'm a little confused with the differing plural forms of `filter`, more so now that there's potentially multiple saved filters.

This may be nit-picky, but I'd recommend maybe naming them something different, `filter` vs. `saved filter` or similar. `commit(SET_FILTER, filters)` is also confusing now.

Reply:
It may just be the usage of `SET_SELECTED_FILTER` as `SET_SAVED_FILTERS` is already a thing.

`SET_SELECTED_SAVED_FILTER`?